packages/blitz/src/bin/cli.ts

```diff
@@ -5,12 +5,16 @@ import chalk from "chalk"
 import {parseSemver} from "../utils/parse-semver"
 
 async function main() {
-  console.log(
-    chalk.yellow(
-      `You are using alpha software - if you have any problems, please open an issue here:
-    https://github.com/blitz-js/blitz/issues/new/choose\n`,
-    ),
-  )
+  const options = require("minimist")(process.argv.slice(2))
+
+  if (options._[0] !== "autocomplete:script" || Object.keys(options).length > 1) {
```

Comment:
```suggestion
  if (options._[0] !== "autocomplete:script") {
```

Unless I'm totally missing something, this second condition is not needed.

Reply:
I wanted to only disable the alpha software message on the command itself, the second argument ensures that when you type `blitz --help autocomplete` or `blitz autocomplete --help` or any other autocomplete command with a flag the warning is still printed.